libr/bin/p/bin_elf.inc

```diff
@@ -715,6 +715,17 @@ static RBinReloc *reloc_convert(struct Elf_(r_bin_elf_obj_t) *bin, RBinElfReloc
 		break;
 		}
 		break;
+	case EM_BPF: switch (rel->type) {
+		case R_BPF_NONE: break;
+		case R_BPF_64_64: r->vaddr += 4; ADD (32, 0);
```

Comment:
```suggestion
		case R_BPF_64_64: r->vaddr += 4; ADD (32, 0); break;
```

Reply:
@trufae The `ADD` macro is defined as follows:

```
#define ADD(T, A) r->type = R_BIN_RELOC_ ## T; r->addend += A; r->additive = rel->mode == DT_RELA; return r
```

And all the other relocs don't use a break either – should I change them too?

Reply:
Correct. there's no need to place a break after a return, but it seems more consistent with a break to me. otherwise it's a little confusing for the reader. (just trying to improve overall code quality, not related to your patch in here), but always feel free to make things more readable if you find them :)

Reply:
Thanks for clarifying. Will fix 🙏🏻